pkg/docker/config/config.go

```diff
@@ -104,15 +125,14 @@ func GetAllCredentials(sys *types.SystemContext) (map[string]types.DockerAuthCon
 
 		// Credential helpers may override credentials from the auth file.
 		for registry, credHelper := range auths.CredHelpers {
-			username, password, err := getAuthFromCredHelper(credHelper, registry)
+			conf, err := getAuthFromCredHelper(credHelper, registry)
```

Comment:
`GetAllCredentials` needs to return the credentials from the `registryies.conf`-configured helpers as well, I think. (@vrothberg ?)